Detectors/TRD/base/include/TRDBase/RecoParam.h

```diff
@@ -0,0 +1,60 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file RecoParam.h
+/// \brief Error parameterizations and helper functions for TRD reconstruction
+/// \author Ole Schmidt
+
+#ifndef O2_TRD_RECOPARAM_H
+#define O2_TRD_RECOPARAM_H
+
+#include <array>
+#include "Rtypes.h"
+
+namespace o2
+{
+namespace trd
+{
+
+class RecoParam
```

Comment:
Do we have some more documentation for this class? Maybe the relevant variables are defined in your thesis, Ole?

Reply:
Yes, this is documented in my thesis. I could add a pointer here to the relevant chapter or just write a more detailed comment on how the tracklet resolution is determined